### PR TITLE
[MAINTENANCE] Set collapse navigation to False

### PR DESCRIPTION
**Pull request by jcampbell**
Signed-off-by: James Campbell <james.p.campbell@gmail.com>

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -18,10 +18,22 @@ communication between teams. Head over to the :ref:`intro` to learn more, or jum
...
+
+.. toctree::
+   :maxdepth: 2
+
+   /guides/tutorials
+   /guides/workflows_patterns
+   /guides/how_to_guides
+
+
+.. toctree::
```
Comment (spbail) [inline docs/index.rst:37]: Two things:
1. Can you explain the change?
2. I'm super confused, I built this locally and still see `community` and `contributing` show up here.

Comment (jcampbell) [reply docs/index.rst:37]: Well, there's "explain the change" and "explain the change"....apparently, this kind of TOC collapsing is a fairly common issue with sphinx themes (?) and there are multiple approaches. With ONE `toctree` you only get ONE section that can be opened at a time. If we break it out like in the experiment here (I'll do it for all of them now that the build shows it works), you can have multiple open at the same time.

Comment (jcampbell) [reply docs/index.rst:37]: ...but WHY it is built like that...no idea.

Comment (jcampbell) [reply docs/index.rst:37]: (at least, I think...I was viewing this as a guess-and-check exercise)

Comment (spbail) [reply docs/index.rst:37]: Oooh okay, that makes sense!

Comment (spbail) [reply docs/index.rst:37]: Makes sense but also, 🤯 just a little.

**Review by spbail — APPROVED** [review]

**Comment by spbail** [comment]
Aah we should merge this

**Pull request merged**